src/integrationTests/communities.test.ts

```diff
@@ -0,0 +1,218 @@
+      },
+      dmKeys: {
+        publicKey:
+          '0bd934b164fdbf09a2675233bd9d5c396ce3a5944f92485c8c98b72ec3148f51',
+        privateKey:
+          '51da400fb7323793604ec204c60f3e1c96b1b3023d2eadc515376e37faf4b9f8',
+      },
+    });
+
+    let communityChannels = new CommunityChannels('F1141EBCF93387E5A28696C5B41E2177');
+
+    communityChannels.currentChannel = 'general';
+
+    const userCsr: UserCsr = await createUserCsr({
+      zbayNickname: 'fgdgfg',
+      commonName: 'ybrmqwsudwxjpzugnl66hx2526nox5nzgmrtzveteud5f7anpjw62zqd',
+      peerId: 'QmXRJb5gdjcrb9VN1UEEnc5VYUHcDppJYykpf5QHWSrGjC',
+      dmPublicKey:
+        '0bd934b164fdbf09a2675233bd9d5c396ce3a5944f92485c8c98b72ec3148f51',
+      signAlg: config.signAlg,
+      hashAlg: config.hashAlg,
+    });
+
+    userCsr.userKey = 'MIGTAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBHkwdwIBAQQgrSJ366ji2AfRNC9BDnViHl5V+A+fnjiUdUuPav3G8rygCgYIKoZIzj0DAQehRANCAAQpI18Np+59RC7UXkAfcl8mu8mrtgAwjv5pN18RmzGw2vV6iHbJ8RrujJZF9Z5rDqGYZAP2UCTyKzWTOC6740RD'
+  
+    identity.userCsr = userCsr;
+    identity.userCertificate =
+      'MIICDTCCAbMCBgF9Lq/VWDAKBggqhkjOPQQDAjASMRAwDgYDVQQDEwdaYmF5IENBMB4XDTIxMTExNzE2MTY1NVoXDTMwMDEzMTIzMDAwMFowSTFHMEUGA1UEAxM+eWJybXF3c3Vkd3hqcHp1Z25sNjZoeDI1MjZub3g1bnpnbXJ0enZldGV1ZDVmN2FucGp3NjJ6cWQub25pb24wWTATBgcqhkjOPQIBBggqhkjOPQMBBwNCAAQpI18Np+59RC7UXkAfcl8mu8mrtgAwjv5pN18RmzGw2vV6iHbJ8RrujJZF9Z5rDqGYZAP2UCTyKzWTOC6740RDo4HCMIG/MAkGA1UdEwQCMAAwCwYDVR0PBAQDAgCOMB0GA1UdJQQWMBQGCCsGAQUFBwMCBggrBgEFBQcDATAvBgkqhkiG9w0BCQwEIgQgFzGXNQSk2IzgcI35nT9GmfKunA1jJHOdV6ZWRhOADhIwFgYKKwYBBAGDjBsCAQQIEwZmZ2RnZmcwPQYJKwYBAgEPAwEBBDATLlFtWFJKYjVnZGpjcmI5Vk4xVUVFbmM1VllVSGNEcHBKWXlrcGY1UUhXU3JHakMwCgYIKoZIzj0EAwIDSAAwRQIhAJmsMNFbttx9i0OMYIzKsALrfCUU8rX70S8Oj2IZp/vWAiA0b+6MR0ANnJkjW6HcClS6K2XHvOizmzQab+8rJNJVag==';
+  
+    const app = await createApp({
+      [StoreKeys.Communities]: {
+        ...new CommunitiesState(),
+        currentCommunity: 'F1141EBCF93387E5A28696C5B41E2177',
+        communities: {
+          ...communitiesAdapter.setAll(communitiesAdapter.getInitialState(), [
+            community,
+          ]),
+        },
+      },
+      [StoreKeys.Identity]: {
+        ...new IdentityState(),
+        identities: {
+          ...identityAdapter.setAll(identityAdapter.getInitialState(), [
+            identity,
+          ]),
+        },
+      },
+  
+        [StoreKeys.PublicChannels]: {
+          ...new PublicChannelsState(),
+          channels: channelsByCommunityAdapter.setAll(
+            channelsByCommunityAdapter.getInitialState(),
+            [communityChannels]
+          ),
+        },
+      
+    });
+
+    await app.runSaga(launchCommunitiesOnStartupSaga).toPromise();
```

Comment:
How this test checks that services were actually launched?

Reply:
launching registrars and communities doesn't change anything in store, so the only way to check at this moment is to run saga, similar what we've had before

Reply:
It waits for response actions from waggle with after successfull registrar and community launch

Reply:
@EmiM does this seem ok to you?

Reply:
If we don't have anything to check then yes. We talked with Bartek some time ago that we could add info to store that community has been launched (a simple bool flag). It would solve this problem.
However I would add it in next iteration, not in this MR.

Reply:
Definitely not this iteration.

Reply:
Let's create an issue for this.